shared_model/validators/transaction_validator.hpp

```diff
@@ -226,6 +227,15 @@ namespace shared_model {
 
      private:
       FieldValidator validator_;
+      mutable int command_counter{0};
```

Comment:
why `mutable`?

Reply:
Because all validation methods are const, and we want to modify counter in them.

Reply:
It could be used in different functions, and I do not think static variable provides any major benefit over class member, and class member expresses intent more clearly.

Reply:
It's possible to encapsulate this var in separate function for that and I don't see any issues here. Moreover it can be helpful in enhancing with the async support